src/main/webapp/WEB-INF/tags/head.tag

```diff
@@ -1,2 +1,3 @@
 <%@tag description="Common head content" pageEncoding="UTF-8"%>
-<!-- This is where common <link> and <script> tags would go. -->
+<link href="../../style.css" rel="stylesheet" type="text/css">
```

Comment:
The href refers to the filepath at _runtime_ (since that's when it's evaluated). At runtime, all the current pages are served from / or /search, so the ../.. is unnecessary. To check what I'm saying, try loading localhost:8080/../../style.css - it'll redirect to localhost:8080/style.css.